### PR TITLE
Add delete device location method

### PR DESCRIPTION
**Pull request by xxuejie**

### CODE REVIEW

File: lib/m2x/device.rb [hunk 1]
```diff
@@ -99,6 +99,13 @@ def update_location(params)
     @client.put("#{path}/location", nil, params, "Content-Type" => "application/json")
   end
 
+  # Delete waypoints in a device's location history by a date range
+  #
+  # https://m2x.att.com/developer/documentation/v2/device#Delete-Location-History
+  def delete_location(params)
```
Comment (fsaravia) [inline lib/m2x/device.rb]: Let's keep the same documentation approach and method signature of `Stream#delete_values!`. Both methods are intended for similar things so keeping consistency among them would look better, I think.

Comment (xxuejie) [inline lib/m2x/device.rb]: Done!

**Comment by fsaravia** [comment]
Thank you! :shipit:

**Comment by xxuejie** [comment]
@fsaravia No problem! Do you want me to merge it or you prefer to do it at your side?

**Comment by fsaravia** [comment]
@xxuejie merge away! I'll craft a release once done.

**Comment by xxuejie** [comment]
Done!

**Pull request merged**